packages/perseus/src/widgets/group/group.test.tsx

```diff
@@ -249,8 +249,8 @@ describe("group widget", () => {
                       "originalIndex": 4,
                     },
                   ],
-                  "countChoices": undefined,
-                  "deselectEnabled": undefined,
+                  "countChoices": false,
+                  "deselectEnabled": false,
```

Comment:
Okay, this is worth pointing out. I have no idea why this changed and I can't figure it out. To me it seemed that these should have been `false` though? So IDK.